### PR TITLE
add 9.3 paper

### PR DESCRIPTION
**Pull request by tjhei**
we might want to wait until it is available online, but here is the PR already.

### CODE REVIEW

File: publications.include [hunk 1]
```diff
@@ -15,48 +15,45 @@
...
+@article{dealII93,
+  title     = {The \texttt{deal.II} Library, Version 9.3},
   author    = {Daniel Arndt and Wolfgang Bangerth and Bruno Blais and
-               Thomas C. Clevenger and Marc Fehling and Alexander V. Grayver and
-               Timo Heister and Luca Heltai and Martin Kronbichler and Matthias Maier and
-               Peter Munch and Jean-Paul Pelteret and Reza Rastak and
-               Ignacio Thomas and Bruno Turcksin and Zhuoran Wang and David Wells},
+               Marc Fehling and Rene Gassm{\"o}ller and
+               Timo Heister and Luca Heltai and Uwe K{\"o}cher
+	       and Martin Kronbichler and Matthias Maier and
```
Comment (bangerth) [inline publications.include:50]: Please adjust these slightly awkwardly indented lines -- these are shown verbatim on the page

File: publications.include [hunk 2]
```diff
@@ -15,48 +15,45 @@
...
+	       and David Wells and Jiaqi Zhang},
   journal   = {Journal of Numerical Mathematics},
-  publisher = {De Gruyter},
-  year      = {2020},
-  volume    = {28},
-  number    = {3},
-  pages     = {131-146},
-  DOI       = {10.1515/jnma-2020-0043},
-  url       = {https://dealii.org/deal92-preprint.pdf}
+  year      = {2021},
```
Comment (bangerth) [inline publications.include:55]:
```suggestion
  year      = {2021, accepted for publication},
```

**Review by bangerth — COMMENTED** [review]
With the minor adjustments, I think this can already be merged now. No need to wait, who knows how long that's going to take.

**Comment by tjhei** [comment]
updated.

**Review by bangerth — APPROVED** [review]

**Pull request merged**